core/src/test/java/com/opensymphony/xwork2/config/providers/EnvsValueSubstitutorTest.java

```diff
@@ -20,16 +20,55 @@
 
 import org.apache.struts2.StrutsInternalTestCase;
 
-
 public class EnvsValueSubstitutorTest extends StrutsInternalTestCase {
 
+    private boolean osIsWindows = false;  // Assume Linux/Unix environment by default
+
+    @Override
+    protected void setUp() throws Exception {
+        super.setUp();
+
+        final String os = System.getProperty("os.name");
```

Comment:
We have `commons-lang3` on the classpath -- IMHO it would be better to use `SystemUtils.IS_OS_WINDOWS` for that check as a one-liner at L:25.

Reply:
Hello @sepe81 .

This PR copies the equivalent code from previous PR#261 (into 2.6/master), and in both cases I had not considered using additional imports/libraries for the Windows OS check.  If you think using `SystemUtils.IS_OS_WINDOWS` from `commons-lang3` is preferable that is certainly doable (in which case it should probably also be done for the same tests in 2.6/master).  

I'm not sure if it's cleaner to incorporate the fix as-is (to mirror 2.6/master more closely), and then follow-up with an "improve/cleanup" version for 2.5.x and master branches.

Please let me know what you think.  :)

Reply:
Hey @JCgH4164838Gh792C124B5 

you are right. In this case it's just kind of cherry-picking. Better we try some improvement exclusively for the master branch. :+1:

Reply:
see https://github.com/apache/struts/pull/272